src/client_backend/triton_c_api/triton_loader.cc

```diff
@@ -1041,10 +1041,30 @@ TritonLoader::GetOutputs(
             &byte_size, &memory_type, &memory_type_id, &userp),
         "inference_response_output_fn_ error");
 
+    std::string data_type{datatype};
```

Comment:
This feels like this should be an std::string_view, but that's not really a big deal, all things considered, in this specific scenario.